tests/e2e.sh

```diff
@@ -162,6 +177,85 @@ assert_dir_exists payments-dev
 assert_dir_exists payments-prod
 grep -q allowPrivilegeEscalation podsecuritypolicy_psp.yaml
 
+helm_testcase "docker_helm_template_undefined_args"
```

Comment:
these are getting too unit-testy IMO. We should have 1 or 2 e2e tests as they're not maintainable in the long run.

Reply:
Removed the unit-testy ones.